src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -1389,6 +1393,13 @@ private static JsonObjectBuilder jsonLicense(DatasetVersion dsv) {
         }
         return licenseJsonObjectBuilder;
     }
+    private static JsonObjectBuilder jsonLatestPublishedVersion(DatasetVersion dsv) {
+        JsonObjectBuilder lpvJsonObjectBuilder = jsonObjectBuilder()
+                .add("id", dsv.getId())
+                .add("versionNumber", dsv.getVersionNumber())
```

Comment:
Note these are returned for the requested version itself on lines 431/432. It would be worthwhile to get feedback from @GPortas as to whether this shortcut (not having to call the api to get the :latest-published version info which has the major/minor version number in it) is needed before changing the code further. If it is, we should probably avoid sending duplicate info when the requested version is the latest published one.